### ISSUE TITLE
Images in slider load on page load, not scroll

### ISSUE THREAD

**Issue by wozniakbe** [eisbehr-/jquery.lazy#132]
I have a few divs that hold collections of images. Two of them are accordion style, and lazy loading works perfectly fine. However, in the slider, there is strange behavior. Say the slider shows 3 images at once. The first 3 images load on page load, even though the slider is far down the page. Then, the next 3 images lazy load when the first 3 images should have loaded up. If you click once right on the slider, the first 3 images don't load (since they were lazy loaded on scroll) and then subsequent right clicks lazy load the images correctly.

In summary, the behavior I am trying to correct is the 3 images loading on page load, and the subsequent 3 loading on scroll. Below is how I am initializing Lazy, the img itself has `class="lazy" data-lazy="image_path"`

`$('img.lazy').Lazy({
            attribute: 'data-lazy',
            chainable: 'false',
            visibleOnly: 'true'
        });`

Let me know if this makes sense, I can try to provide more info but can't post too much of the code. Thanks!

**Comment by eisbehr-** [comment]
It's hard to tell where the problem is without seeing the code. Can't give an answer for this here by your description.

**Comment by wozniakbe** [comment]
Issue ended up being due to attribute name, had to change that and it is working. Not sure where the conflict was from but it's good.
